src/main/java/core/basesyntax/DrawArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawArea {
```

Comment:
Інтерфейс Drawable може імплементувати будь який об'єкт, що можна відобразити. Area тут ні до чого і саме тому ми відокремили два незалежні інтерфейси.
```suggestion
public interface Drawable {
```